docs/content/ingestion/index.md

```diff
@@ -71,6 +72,7 @@ An example dataSchema is shown below:
 | dataSource | String | The name of the ingested datasource. Datasources can be thought of as tables. | yes |
 | parser | JSON Object | Specifies how ingested data can be parsed. | yes |
 | metricsSpec | JSON Object array | A list of [aggregators](../querying/aggregations.html). | yes |
+| rollup | Boolean | rollup or not | no (default == true) |
```

Comment:
From that I know of the Druid commiter people they will not like the "default to true" settings. Things should always default to false or so they say :-p. Now the 'simple fix' is to change this setting to (say) `disableRollup` and have it default to `false`. **But** I think we should have a conversation about having no rollup be the default in Druid.

'No rollup' is how most DBs out there operate. It would make Druid much easier to evaluate because you would not need to have to think about rollup from day 0.

I would love to get the Druid people @xvrl , @gianm , @cheddar , @fjy , @himanshug , @drcrallen and anyone I am forgetting to comment on possibly making Druid have no rollup by default.